index.html

```diff
@@ -0,0 +1,234 @@
+            />
+          </a>
+
+          <div class="nav__mobile">
+            <input type="checkbox" id="check" class="nav__mobile--check" />
+            <label for="check" class="nav__mobile--label">
+              <span class="nav__mobile--icon">&nbsp;</span>
+            </label>
+
+            <div class="nav__mobile--sidebar">
+              <div class="nav__mobile--header">
+                <span>Quick Links</span>
+              </div>
+              <ul class="nav__mobile--menu">
+                <li class="nav__mobile--list">
+                  <a href="about.html#aboutto" class="nav__mobile--links">About</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#program" class="nav__mobile--links">Program</a>
+                </li>
+                <li class="nav__mobile--list">
+                    <a href="#" class="nav__mobile--links">Join</a>
+                  </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#sponsors" class="nav__mobile--links">Sponsor</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#speakers" class="nav__mobile--links">Speakers</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="about.html#check" class="nav__mobile--links">Africa Rise</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <ul class="nav__menu">
+            <li class="nav__list">
+              <a href="about.html#aboutto" class="nav__links">About</a>
+            </li>
+            <li class="nav__list">
+              <a href="#program" class="nav__links">Program</a>
+            </li>
+            <li class="nav__list">
+                <a href="#" class="nav__links">Join</a>
+              </li>
+            <li class="nav__list">
+              <a href="index.html#sponsors" class="nav__links">Sponsor</a>
+            </li>
+            <li class="nav__list">
+              <a href="#speakers" class="nav__links">Speakers</a>
+            </li>
+            <li class="nav__list">
+              <a href="about.html#check" class="nav__links">Africa Rise</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="intro">
+      <div class="header__content">
+        <div class="heading">
+          <p class="heading-sub">
+            "Lets Connect and Build Africa."
+          </p>
+          <h1 class="heading-primary">
+            AFRICA YOUTH SUMMIT 2022
+          </h1>
+          <span class="heading-theme">
+            Equipping Africa's Youth for Continental Development.
+          </span>
+          <span class="heading-date">06.06.2022(MON) ~ 09(THUR)</span>
+          <span class="heading-venue">
+            @Kololo memorial independence grounds, Kampala Uganda
+          </span>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Please make sure the left alignment is consistent between contiguous objects. 
![image](https://user-images.githubusercontent.com/52635378/170832531-899d5429-df93-4dd0-b95b-eb085fa7f243.png)